Verify.md

```diff
@@ -0,0 +1,81 @@
+# Verify Contracts
+
+For each contract whose address appears in migration.json (and therefore is a "migrated contract") we send the contract's solidity code to etherscan.io which will attempt to "verify" that the given code corresponds to the bytecodes deployed at the address given in migration.json.
+
+**Note** Contract validation via this script currently does not work on Windows (truffle-flattener generates invalid flattened .sol files).  The fix for that is at this writing in PR at truffle-flattener.
+
+## Setup
+1. In the project root, create an ".env" file (or use the existing one) that contains:
+```
+APIKEY=[API key from etherscan.io]
```

Comment:
please add a link to etherscan api key ..how too.
https://etherscan.io/apis

Reply:
done